packages/docusaurus-plugin-content-blog/src/plugin-content-blog.d.ts

```diff
@@ -73,6 +73,13 @@ yarn workspace v1.22.19image` is a collocated image path, this entry will be the
      * Unknown keys are allowed, so that we can pass custom fields to authors,
      * e.g., `twitter`.
      */
+    socials?: {
+      twitter: string;
+      linkedin: string;
+      github: string;
+      stackoverflow: string;
+      [key: string]: string;
```

Comment:
I'd rename `key` to `customAuthorSocialPlatform` to make this more explicit and self-documenting.